mne/epochs.py

```diff
@@ -2164,6 +2166,9 @@ def __init__(self, data, info, events, tmin=0, event_id=None,
         if len(info['ch_names']) != data.shape[1]:
             raise ValueError('Info and data must have same number of '
                              'channels.')
+        if events is None:
+            n_trial = len(data)
+            events = np.c_[range(n_trial), [0] * n_trial, [1] * n_trial]
```

Comment:
not sure if this goes well with pytho 3.
why not standard numpy:

```
np.c_[np.arange(n_events), np.zeros(n_events), np.ones(n_events)]
```
Also we usually don't call it trials, maybe events or epochs?